client/webserver/site/src/css/market.scss

```diff
@@ -56,6 +56,34 @@ div.popupforms {
   z-index: 100;
 }
 
+div.market-orders-table-container {
+  @media (min-width: 992px) {
+    position: absolute;
+    left: 0;
+    top: 0;
+    bottom: 0;
+    right: 0;
+  }
+
+  @media (max-width: 991px) {
```

Comment:
I feel like there should be a way to use the break points defined in application.scss, no?  Like `media-breakpoint-down(md)` and `media-breakpoint-up(lg)`

https://github.com/decred/dcrdex/blob/90b1d9289eb55e74d59840f3a856d44a313559df/client/webserver/site/src/css/application.scss#L17-L21

If doing so makes things more complex, then this is fine, but it seems a bit fragile because we risk changing one but not the other.

Reply:
With `@media-breakpoint-down` and `@media-breakpoint-up` that are different from what we are using, but what I have now has the same effect.